pkgs/applications/version-management/git-and-tools/git-codereview/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage {
```

Comment:
`buildGoPackage` without `deps.nix`? Plus, they seem to use go modules, see https://github.com/golang/review/blob/master/go.mod

Reply:
They don't have any external dependencies, so I'm unsure what you're expecting the `deps.nix` to contain.

Reply:
OK, so no `deps.nix`. But have you tried using `buildGoModule`?

Reply:
`buildGoPackage` is considered deprecated, just as other Go dependency management systems are considered deprecated by Google.

Reply:
> `buildGoPackage` is considered deprecated

I'm not seeing any particular proof for this claim, and it still seems perfectly fine for building Go packages with no un-vendored external dependencies.

Reply:
See https://nixos.org/nixpkgs/manual/#ssec-go-legacy .

Reply:
Actually, `buildGoModule` is less preferable since it abuses fixed-output derivations https://github.com/NixOS/nix/issues/2270